lib/fastly_nsq/listener.rb

```diff
@@ -16,6 +17,7 @@ def go(run_once: false)
 
       loop do
         next_message do |message|
+          preprocessing.call(message.body) if preprocessing
```

Comment:
Was going to say we might want to check `if preprocessing.is_a?(Proc)` but as long as it's documented I don't think that's necessary.